Cargo.toml

```diff
@@ -8,6 +8,7 @@ categories = ["visualization", "development-tools::build-utils"]
 repository = "https://github.com/mersinvald/aquamarine"
 edition = "2018"
 license = "MIT"
+include = [ "doc/**/*" ]
```

Comment:
It's not very important, but since there's just one file copied in `build.rs`, I think there should be just one file listed in the `include` too

Reply:
Gotcha